.github/workflows/ort.yml

```diff
@@ -158,14 +159,28 @@ jobs:
               with:
                 folder_path: "${{ github.workspace }}/glide-core"
 
+            ### Java ###
+
+            - name: Set up JDK 11
```

Comment:
I think JDK11 is setup already above (installation for ORT).  I mean, it doesn't hurt, but I don't think this step does anything.